components/console/logger.rst

```diff
@@ -85,6 +85,8 @@ through the second parameter of the :class:`Symfony\\Component\\Console\\Console
 constructor::
 
     // ...
+    use Psr\Log\LogLevel;
+    // ...
```

Comment:
I don't have hard feelings about this, but imho the duplicated placeholder doesn't add any value. People will soon realise anyway that the example is not complete. :)